src/ui/marker.js

```diff
@@ -216,9 +216,13 @@ export default class Marker extends Evented {
     }
 
     /**
-     * Attaches the marker to a map
+     * Attaches a `Marker` to a `Map` object.
```

Comment:
just nit picking here, but addTo doesn't add any `Marker`, it only adds `this` `Marker`, so should that be:
_attaches **the** `Marker` to_...

maybe I'm overthinking it.

Reply:
> maybe I'm overthinking it.

@andrewharvey Not at all! That distinction makes it a lot clearer - I will update the PR. Thank you!